### PR TITLE
HW-67510 : Update UI SDK localizable.strings

### PR DESCRIPTION
**Pull request by ddhar-hw**
HW-67510 : Update UI SDK localizable.strings

### CODE REVIEW

File: Common/Sources/Localization/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -165,6 +165,28 @@
...
+"prepaid_card_chargeback_refund_reversal" = "Chargeback Refund Reversal";
+"prepaid_card_chargeback_reversal" = "Chargeback Reversal";
+"prepaid_card_commission_or_fee" = "Fee / Commission";
+"prepaid_card_debit_transfer" = "Debit/Transfers";
+"prepaid_card_document_request_fee" = "Requests for copies of documents";
+"prepaid_card_emergency_cash" = "Emergency money";
+"prepaid_card_emergency_card" = "Emergency card";
+"prepaid_card_income" = "Income";
+"prepaid_card_load_fee" = "Fee for load pan";
+"prepaid_card_overdue_payment_interest" = "Interest on overdue payment";
```
Comment (jsaini-hw) [inline Common/Sources/Localization/en.lproj/Localizable.strings:189]: You have to add these as enum cases in Core SDK as well

File: Common/Sources/Localization/en.lproj/Localizable.strings [hunk 2]
```diff
@@ -165,6 +165,28 @@
...
+"prepaid_card_chargeback_refund" = "Chargeback Refund";
+"prepaid_card_chargeback_refund_reversal" = "Chargeback Refund Reversal";
+"prepaid_card_chargeback_reversal" = "Chargeback Reversal";
+"prepaid_card_commission_or_fee" = "Fee / Commission";
+"prepaid_card_debit_transfer" = "Debit/Transfers";
+"prepaid_card_document_request_fee" = "Requests for copies of documents";
+"prepaid_card_emergency_cash" = "Emergency money";
+"prepaid_card_emergency_card" = "Emergency card";
+"prepaid_card_income" = "Income";
+"prepaid_card_load_fee" = "Fee for load pan";
```
Comment (jsaini-hw) [inline Common/Sources/Localization/en.lproj/Localizable.strings:188]: what is pan?

Comment (ddhar-hw) [reply Common/Sources/Localization/en.lproj/Localizable.strings:188]: Thats what we have in our PROD db

File: Common/Sources/Localization/en.lproj/Localizable.strings (hunk 1 above)
Comment (ddhar-hw) [reply Common/Sources/Localization/en.lproj/Localizable.strings:189]: https://github.com/hyperwallet/hyperwallet-ios-sdk/pull/96

File: Common/Sources/Localization/en.lproj/Localizable.strings (hunk 2 above)
Comment (ddhar-hw) [reply Common/Sources/Localization/en.lproj/Localizable.strings:188]: I believe it should be plan - but need to confirm this with PO

**Review by peter-joseph — DISMISSED** [review]

File: Common/Sources/Localization/en.lproj/Localizable.strings (hunk 2 above)
Comment (peter-joseph) [reply Common/Sources/Localization/en.lproj/Localizable.strings:188]: PAN = Primary Account Number and that content is valid

File: Cartfile [hunk 3]
```diff
@@ -1,2 +1,2 @@
-github "hyperwallet/hyperwallet-ios-sdk" "development"
+github "hyperwallet/hyperwallet-ios-sdk" "task/HW-66963-ppc-transactions"
```
Comment (jsaini-hw) [inline Cartfile:1]: revert these changes once @malaw-hw merges Core SDK PR to development branch

**Review by jsaini-hw — APPROVED** [review]

**Review by malaw-hw — APPROVED** [review]
